test/intl402/NumberFormat/prototype/toStringTag/prop-desc.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.numberformat.prototype-@@tostringtag
+description: >
+  Tests that Intl.NumberFormat.prototype[@@toStringTag] has the required attributes.
+includes: [propertyHelper.js]
+---*/
+
+verifyProperty(Intl.NumberFormat, Symbol.toStringTag, {
```

Comment:
This should be checking Intl.NumerFormat.prototype

Reply:
This should be checking the value as well

Reply:
Oops, on it.